Bio/Phylo/_utils.py

```diff
@@ -640,3 +640,195 @@ def draw_clade(clade, x_start, color, lw):
 
     if do_show:
         plt.show()
+
+
+def build_ete3tree(tree, ete_root):
```

Comment:
How about `to_ete3`, similar to `to_networkx`?

Reply:
yeah, it's better